src/search/search.component.ts

```diff
@@ -11,20 +11,6 @@ import {
 import { NG_VALUE_ACCESSOR, ControlValueAccessor } from "@angular/forms";
 import { I18n } from "../i18n/i18n.module";
 
-/**
- * Used to emit changes performed on search components.
- */
-export class SearchChange {
```

Comment:
We can't remove this just yet ... we can definitely mark it deprecated however :+1: